pandas/core/groupby/groupby.py

```diff
@@ -1747,6 +1747,35 @@ def last(x: Series):
             npfunc=last_compat,
         )
 
+    @final
+    @doc(_groupby_agg_method_template, fname="only unique", no=False, mc=-1)
+    def only_unique(self, numeric_only: bool = False, min_count: int = -1):
+        def only_unique_compat(obj: FrameOrSeries, axis: int = 0):
+            def only_unique(x: Series):
+                """Helper function for only unique item."""
+                arr = set(x)
+                if not arr:
+                    return np.nan
```

Comment:
This line is a relic from when I was dropping NA values, but I don't think you can reach it anymore. I'll leave it until I get some feedback though.